content/en/load-balancer.md

```diff
@@ -5,20 +5,12 @@ category: concept
 tags: ["infrastructure", "networking", ""]
 ---
 
-A load balancer is a tool that efficiently distributes incoming requests among multiple instances of an application. 
-Take a [microservice](/microservices/) architecture for example, where each service can be [scaled horizontally](/horizontal-scaling/). 
-A load balancer sits in front of a scaled microservice and ensures that no one instance gets the bulk of the requests.
-Load balancers can be software or hardware-based.
+A load balancer is a crucial component in computer networks that distributes incoming requests across multiple servers. By doing so, it prevents any single server from becoming overwhelmed, ensuring consistent and reliable performance for users accessing websites or applications.
```

Comment:
I would suggest to use instances of an application instead of servers here and in the following content.

Reply:
To be super honest, I don't find the new text providing more clear information. I find the old text better.